### DISCUSSION TITLE
[feature request] introduce lists to the bookmark feature

### DISCUSSION THREAD

**Discussion by cmalloc** [mastodon/mastodon#21734] [General]
Bookmarking toots is a very cool feature that I am using more and more. I think it would be great to have the ability to setup different lists within the bookmarks. So if you are into different topics like travelling, cycling, etc. you can sort the toots you bookmark in lists per topic, With a growing amount of bookmarks this would improve overview.

**Comment by shaedrich** [comment]
Issue: #28409
